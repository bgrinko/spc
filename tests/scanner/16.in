begin
end
function
var
a, b, c, d, e
/* fdsfdsg
dfgdfg
dfgdfg