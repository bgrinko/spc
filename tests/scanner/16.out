1	5	NONE		begin
3	3	NONE		end
5	8	NONE		function
7	3	NONE		var
9	1	NONE		a
9	2	NONE		,
9	4	NONE		b
9	5	NONE		,
9	7	NONE		c
9	8	NONE		,
9	10	NONE		d
9	11	NONE		,
9	13	NONE		e
Error! Line: 11 Pos: 2 Msg: Expression expected
